Earthfile

```diff
@@ -66,16 +69,15 @@ all-arm:
   ARG SECURITY_SCANS=true
   BUILD --platform=linux/arm64 +base-image --MODEL=rpi64
   IF [ "$SECURITY_SCANS" = "true" ]
-      BUILD --platform=linux/arm64 +image-sbom --MODEL=rpi64
-      BUILD --platform=linux/arm64 +trivy-scan --MODEL=rpi64
-      BUILD --platform=linux/arm64 +grype-scan --MODEL=rpi64
+      BUILD --platform=linux/arm64 +image-sbom --MODEL=rpi64 --ARCH=arm64
```

Comment:
could we infer the arch directly via what earthly sets in env vars that way we dont have to "manually" pass them and will work everywhere? 

`TARGETARCH` can be `arm, amd64, arm64`

Reply:
Good call